### PR TITLE
RM - Monthly survey

### PR DESCRIPTION
**Pull request by anvmn**
#596

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2023-01-19 23-53

**Comment by anvmn** [comment]
Peek 2023-01-22 10-48

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Backend/Endpoints.elm [hunk 1]
```diff
@@ -1207,3 +1210,20 @@ ncdHbA1cTestEndpoint : ReadWriteEndPoint Error NCDHbA1cTestId NCDHbA1cTest NCDHb
...
+resilienceSurveyEndpoint =
+    swEndpoint "nodes/resilience_survey" decodeResilienceSurvey
+        |> withValueEncoder (object << encodeResilienceSurvey)
+        |> withParamsEncoder encodeResilienceSurveyParams
+
+
+encodeResilienceSurveyParams : Maybe NurseId -> List ( String, String )
+encodeResilienceSurveyParams params =
+    case params of
+        Just id ->
```
Comment (mariano-dagostino) [inline client/src/elm/Backend/Endpoints.elm:1225]: `Just uuid`

**Review by mariano-dagostino — APPROVED** [review]
@anvmn Looks good, just one minor improvement if you want.

File: client/src/elm/Backend/Endpoints.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Backend/Endpoints.elm:1225]: We always use `id` for IDs, no matter if it's Int, String or UUID.

**Pull request merged**